l10n_eu_nace/wizards/nace_import.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
```

Comment:
`<openerp><data>` to replace by `<odoo>`

Reply:
Dayum

Reply:
... :D